### PR TITLE
use jmh for benchmarks

### PR DESCRIPTION
**Pull request by RacoonDog**

### CODE REVIEW

**Comment by MineGame159** [comment]
You shouldn't be parsing, compiling and initializing the Starscript instance inside the benchmark.

**Comment by RacoonDog** [comment]
> You shouldn't be parsing, compiling and initializing the Starscript instance inside the benchmark.

oh my bad
github fucked up because i already had a fork and i found it easier just to delete everything and rewrite, guess i forgot about that

File: src/test/java/meteordevelopment/starscript/Benchmark.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        for (int i = 0; i < iterations; i++) {
+            result = String.format(formatSource, 59.68223);
+        }
+
+        return result;
+    }
+
+    @org.openjdk.jmh.annotations.Benchmark
+    public String starscript() {
```
Comment (MineGame159) [inline src/test/java/meteordevelopment/starscript/Benchmark.java:63]: Oh also forgot to mention this before but IIRC the individual benchmarks can take the `Blackhole` argument which is better than just assigning to a variable.

Comment (RacoonDog) [reply src/test/java/meteordevelopment/starscript/Benchmark.java:63]: should I keep the `toString()` in the starscript benchmark?

Comment (MineGame159) [reply src/test/java/meteordevelopment/starscript/Benchmark.java:63]: Uhhh probably should leave it there to match the output of format

**Pull request merged**